server/controllers/BookController.js

```diff
@@ -52,23 +52,29 @@ class BookController {
 */
   static async fetchBooks(request, response) {
     let { page, limit } = request.query;
+    const {
+      title, author, tag, keyword
+    } = request.query;
+    const standardQueries = title || author || tag;
     page = page || 1;
     limit = limit || 10;
+    const queryFilter = filter(title, tag, author, keyword);
     try {
-      const countResults = await Book.findAndCountAll();
+      if (keyword && standardQueries) return responseMessage(response, 400, { status: 'failure', message: 'keyword param cannot be used with title, author or tag' });
```

Comment:
this message is cool, but will the user understand what param means,  look for a more descriptive word instead of params

Reply:
I feel you can simply take away the **param**

Reply:
Done